bin/move_it_mirror.py

```diff
@@ -43,22 +43,22 @@ class MoveItMirror(MoveItBase):
 
     def __init__(self, cmd_args):
         """Set up the mirror."""
-        super(MoveItMirror, self).__init__(cmd_args, "mirror")
-        self.cache_lock = Lock()
         LOGGER.info("Starting publisher on port %s.", str(cmd_args.port))
-        self.pub = Publisher("tcp://*:" + str(cmd_args.port), "move_it_mirror")
+        publisher = Publisher("tcp://*:" + str(cmd_args.port), "move_it_mirror")
+        super(MoveItMirror, self).__init__(cmd_args, "mirror", publisher=publisher)
```

Comment:
This looks nearly identical to the server version, can it be factorize into the base class? or a new intermediate class?

Reply:
I moved the creation to a function defined in `trollmoves.move_it_base`. It can't be created within the base class as we need to recycle it when config is reloaded.